util/expensivequery/expensivequery.go

```diff
@@ -73,12 +82,213 @@ func (eqh *Handle) Run() {
 				}
 			}
 			threshold = atomic.LoadUint64(&variable.ExpensiveQueryTimeThreshold)
+			eqh.oomKillerAlert()
```

Comment:
We can check
``` go
if alert := config.GetGlobalConfig().Performance.ServerMemoryAlert; alert == 0 || alert == 1 || eqh.record.oomRecordErr != nil {
   			eqh.oomKillerAlert()

}
```
here

Reply:
Addressed.  I think the check should be in the initialization part.